### DISCUSSION TITLE
How to get 8 vertices of bounding box

### DISCUSSION THREAD

**Discussion by lovecoding2023** [pyvista/pyvista#2710] [Q&A]
Hi Great pyvista team:

I am trying to create three point labels for the bounding box and display the size text. How can I get the 8 vertices of the bounding box? Or is there an easier way? thanks!

**Comment by nimafo** [comment]
Theoretically, you can infer the 8 points of the bounding box from the two bounds, given that this is not axis-aligend bounding box. 
So, assuming you have the bounding box of the "action figure" database from the following line of code in the form of `(xmin, xmax, ymin, ymax, zmin, zmax)`, 
the other 6 points will be as follows:
point name, X, Y, Z
1,xmin, ymax, zmin
2,xmin,ymax,zmax
3,xmax,ymin,zmin
4,xmax,ymin,zmax
5,xmax,ymax,zmin


```
import pyvista as pv
from pyvista import examples
exmp = examples.download_action_figure() #import example 
bounds = exmp.bounds #get bounds
```
